cpp/arcticdb/storage/mongo/mongo_client.cpp

```diff
@@ -310,31 +360,28 @@ storage::KeySegmentPair MongoClientImpl::read_segment(const std::string &databas
     auto database = client->database(database_name); //TODO maybe cache
     auto collection = database[collection_name];
 
-    try {
-        ARCTICDB_SUBSAMPLE(MongoStorageReadFindOne, 0)
-        auto stream_id = variant_key_id(key);
-        if(StorageFailureSimulator::instance()->configured())
-            StorageFailureSimulator::instance()->go(FailureType::READ);
+    ARCTICDB_SUBSAMPLE(MongoStorageReadFindOne, 0)
+    auto stream_id = variant_key_id(key);
+    if(StorageFailureSimulator::instance()->configured())
+        StorageFailureSimulator::instance()->go(FailureType::READ);
```

Comment:
(Personal preference) I believe that wrapping if/else/for/while bodies in `{}` even for one liners is more readable.